### PR TITLE
[protocol] Update unit-cloud protocol according to the latest changes

### PR DESCRIPTION
**Pull request by DmytroRozzuvailo**

### CODE REVIEW

File: api/cloudprotocol/log.go [hunk 1]
```diff
@@ -64,6 +73,7 @@ type PushLog struct {
 	PartsCount  uint64     `json:"partsCount,omitempty"`
 	Part        uint64     `json:"part,omitempty"`
 	Content     []byte     `json:"content,omitempty"`
+	Status      string     `json:"status"`
```
Comment (al1img) [inline api/cloudprotocol/log.go:76]: Add status consts

File: api/cloudprotocol/unitstatus.go [hunk 2]
```diff
@@ -148,20 +149,28 @@ type InstanceStatus struct {
...
 type LayerStatus struct {
-	LayerID   string     `json:"layerId"`
+	ID        string     `json:"id"`
 	Digest    string     `json:"digest"`
 	Version   string     `json:"version"`
 	Status    string     `json:"status"`
 	ErrorInfo *ErrorInfo `json:"errorInfo,omitempty"`
 }
 
+// ComponentItemStatus component status.
```
Comment (al1img) [inline api/cloudprotocol/unitstatus.go:159]: Why?

File: api/cloudprotocol/envvars.go [hunk 3]
```diff
@@ -36,14 +36,14 @@ const (
 // EnvVarsInstanceInfo struct with envs and related service and user.
 type EnvVarsInstanceInfo struct {
 	InstanceFilter
-	EnvVars []EnvVarInfo `json:"envVars"`
+	Variables []EnvVarInfo `json:"variables"`
 }
 
 // EnvVarInfo env info with id and time to live.
 type EnvVarInfo struct {
-	ID       string     `json:"id"`
```
Comment (al1img) [inline api/cloudprotocol/envvars.go:44]: Why ID is remove?

Comment (DmytroRozzuvailo) [reply api/cloudprotocol/envvars.go:44]: it is renamed to name according to public protocol

File: api/cloudprotocol/unitstatus.go (hunk 2 above)
Comment (DmytroRozzuvailo) [reply api/cloudprotocol/unitstatus.go:159]: component may have several statuses on different nodes that's why we have list of statuses - ComponentItemStatus

File: api/cloudprotocol/log.go (hunk 1 above)
Comment (DmytroRozzuvailo) [reply api/cloudprotocol/log.go:76]: what do you mean?

File: api/cloudprotocol/unitstatus.go [hunk 4]
```diff
@@ -130,7 +131,7 @@ type NodeInfo struct {
 
 // ServiceStatus service status.
 type ServiceStatus struct {
-	ServiceID string     `json:"serviceId"`
```
Comment (al1img) [inline api/cloudprotocol/unitstatus.go:133]: Do not change core name, you can change json tag only.

File: api/cloudprotocol/unitstatus.go [hunk 5]
```diff
@@ -148,7 +149,7 @@ type InstanceStatus struct {
 
 // LayerStatus layer status.
 type LayerStatus struct {
-	LayerID   string     `json:"layerId"`
+	ID        string     `json:"id"`
```
Comment (al1img) [inline api/cloudprotocol/unitstatus.go:152]: ditto

File: api/cloudprotocol/monitoring.go [hunk 6]
```diff
@@ -47,21 +47,21 @@ type MonitoringData struct {
...
 	Disk       []PartitionUsage `json:"disk"`
+	Timestamp  time.Time        `json:"timestamp"`
 }
 
 // NodeMonitoringData node monitoring data.
 type NodeMonitoringData struct {
 	MonitoringData
-	NodeID           string                   `json:"nodeId"`
-	Timestamp        time.Time                `json:"timestamp"`
-	ServiceInstances []InstanceMonitoringData `json:"serviceInstances"`
```
Comment (al1img) [inline api/cloudprotocol/monitoring.go:57]: Also leave `ServiceInstances` for core update only json tag. But `items` looks ambiguous.

File: api/cloudprotocol/unitstatus.go (hunk 4 above)
Comment (DmytroRozzuvailo) [reply api/cloudprotocol/unitstatus.go:133]: okay

File: api/cloudprotocol/unitstatus.go (hunk 5 above)
Comment (DmytroRozzuvailo) [reply api/cloudprotocol/unitstatus.go:152]: okay

**Review by mlohvynenko — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

File: api/cloudprotocol/unitstatus.go [hunk 7]
```diff
@@ -157,11 +158,13 @@ type LayerStatus struct {
 
 // ComponentStatus component status.
 type ComponentStatus struct {
-	ComponentID   string     `json:"componentId"`
-	ComponentType string     `json:"componentType"`
-	Version       string     `json:"version"`
-	Status        string     `json:"status"`
-	ErrorInfo     *ErrorInfo `json:"errorInfo,omitempty"`
+	ComponentID   string          `json:"id"`
```
Comment (mykola-kobets-epam) [inline api/cloudprotocol/unitstatus.go:161]: May be better keep it as is componentId

**Review by mykola-kobets-epam — APPROVED** [review]

File: api/cloudprotocol/unitstatus.go (hunk 7 above)
Comment (DmytroRozzuvailo) [reply api/cloudprotocol/unitstatus.go:161]: we did a changes in all changes - id of component or jayer or service into their statuses agreed to use 'id'. not sure that it need to be reverted - it is json representation

Comment (mykola-kobets-epam) [reply api/cloudprotocol/unitstatus.go:161]: ok, good.

**Pull request merged**